### DISCUSSION TITLE
Update and deploy doesnt starts

### DISCUSSION THREAD

**Discussion by iakzs** [community/community#49037] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hey, i was going to update and deploy my html but it doesnt start and at 5 minutes later, i get a startup failure. how i can fix this?

**Comment by Tchekda** [comment, marked as answer]
Seems like all Github actions are down.. Same here, even on a private runner

**Reply by iakzs** [reply]
ooh thats why, thanks tchekda

**Reply by Tchekda** [reply]
https://www.githubstatus.com/incidents/qfby3shb6r3w
